remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/FirebaseRemoteControllerSyncer.java

```diff
@@ -49,27 +126,30 @@ public FirebaseRemoteControllerSyncer() {
    * happen in practice though, since variables in remote controllers are tied to a single instance
    * of the app (one specific device running the app).
    */
-  private void syncVariableToRemoteController (Variable variable) {
-    // TODO(miguely): write controller sync logic
+  private synchronized void syncVariableToRemoteController (StoredVariable variable) {
+    if (syncing) {
+      reference.child(variable.getKey()).setValue(variable);
+    }
   }
 
   /**
    * Clears all the data in a remote.
    */
   private void clearVariablesInRemoteController() {
-    // TODO(miguely): write controller sync logic
+    reference.removeValue();
```

Comment:
is this method still necessary? You can just inline `reference.removeValue()` (you actually call removeValue() twice in some cases)